airdrop/application/handlers/airdrop_handlers.py

```diff
@@ -68,3 +68,16 @@ def airdrop_window_user_details(event, context):
         response,
         cors_enabled=True,
     )
+
+
+@exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger)
+def airdrop_window_claims(event, context):
+    logger.info(f"Got Airdrops Window Claims Events {event}")
+    status, response = AirdropServices(
+    ).airdrop_window_claims(request(event))
+    return generate_lambda_response(
+        status.value,
+        status.phrase,
+        response,
+        cors_enabled=True,
```

Comment:
do we need to enable cors globally?
can we just whitelist the frontend dapp domains?